src/components/proofs/EmailProof.tsx

```diff
@@ -107,10 +125,11 @@ export default function () {
     setError(undefined)
     try {
       if (secret) await ProofStore.generateEmail(domain, secret)
+      resetEmail(true)
     } finally {
       setLoading(false)
       setOpen(false)
-      setEmail(undefined)
+      resetEmail()
```

Comment:
The user entered email correctly, but copied the token wrongly, he can re-use this saved domain and try to copy-paste the token again